Python/Product/Profiling/Profiling/ProfiledProcess.cs

```diff
@@ -128,7 +128,7 @@ sealed class ProfiledProcess : IDisposable {
             using (var p = ProcessOutput.RunHiddenAndCapture(perfCmdPath, "/waitstart")) {
                 p.Wait();
                 if (p.ExitCode != 0) {
-                    throw new InvalidOperationException("Starting perf cmd failed{0}{0}Output:{0}{1}{0}{0}Error:{0}{2}".FormatUI(
+                    throw new InvalidOperationException(Strings.StartPerfCmdError.FormatUI(
```

Comment:
Newlines can be explicitly entered into the resource, so we don't need to use {0} to insert them.